MUICore/muicore/mui_server.py

```diff
@@ -27,6 +32,21 @@
 
 from .native_utils import parse_native_arguments
 from .evm_utils import setup_detectors_flags
+from manticore.utils.log import CallbackStream
+
+
+class ManticoreWrapper:
+    def __init__(self, mcore_object, mthread):
+        self.uuid = uuid.uuid4().hex
+        self.manticore_object = mcore_object
+        self.thread = mthread
+        self.log_queue = collections.deque(maxlen=5000)
```

Comment:
The `5000` number is just based on experimentation? Any idea how many messages are queued on average?

Reply:
it depends on the level of verbosity set as well as how often one calls GetMessageLog I suppose, but anecdotally with -v and -vv I've never maxed out.

but really the reason I used maxlen=5000 is because manticore itself uses a log queue with maxlen=5000 (implemented as part of the TUI/mc ansible project) https://github.com/trailofbits/manticore/blob/904f33667024de80a7b66dee618ad085ca21c0da/manticore/core/manticore.py#L103

Reply:
Thank you for the explanation! I thought that `5000` number looked familiar 👍